IPA.Loader/Loader/PluginLoader.cs

```diff
@@ -83,9 +83,37 @@ internal static void YeetIfNeeded()
 
         private static readonly Regex embeddedTextDescriptionPattern = new(@"#!\[(.+)\]", RegexOptions.Compiled | RegexOptions.Singleline);
 
+        public static string[] LoadFilesRecursively(string folderPath, string fileName)
+        {
+            var dlls = new List<string>();
+
+            dlls.AddRange(Directory.GetFiles(folderPath, fileName));
+
+            foreach (var subfolder in Directory.GetDirectories(folderPath))
+            {
+                dlls.AddRange(LoadFilesRecursively(subfolder, fileName));
+            }
+
+            return dlls.ToArray();
+        }
+
+        public static string[] LoadDirectoriesRecursively(string folderPath)
+        {
+            var directories = new List<string>();
+
+            foreach (var subfolder in Directory.GetDirectories(folderPath))
+            {
+                directories.AddRange(LoadDirectoriesRecursively(subfolder));
+            }
+
+            directories.Add(folderPath);
+
+            return directories.ToArray();
+        }
+
         internal static void LoadMetadata()
         {
-            string[] plugins = Directory.GetFiles(UnityGame.PluginsPath, "*.dll");
+            string[] plugins = LoadFilesRecursively(UnityGame.PluginsPath, "*.dll");

```

Comment:
Why not use `Directory.GetFiles` with the overload that supports search options?